tox.ini

```
@@ -1,135 +1,129 @@
 [tox]
 isolated_build = True
-envlist = bandit, safety, check-copyright, black-check, isort-check, vulture, flake8, mypy, pylint, py3.8, py3.7, docs
+envlist = bandit, safety, check-copyright, black-check, isort-check, vulture, flake8, mypy, docs, py3{8,9,10,11}
```

```suggestion
envlist = bandit, safety, check-copyright, black-check, isort-check, vulture, flake8, mypy, docs, pylint, py3{8,9,10,11}
```

Pylint gave lots of errors, so I deleted it for the moment. However, we might want to fix them all and have it here again.

---

I think we can fix the pylint issues in a specific PR